test/run.sh

```diff
@@ -100,7 +107,7 @@ sleep 2
 echo "Creating kind cluster named '$CLUSTER_NAME'"
 # Kind image with Kubernetes v1.22.compatible with kind v0.11.1
 kind create cluster --image=kindest/node@sha256:100b3558428386d1372591f8d62add85b900538d94db8e455b66ebaf05a3ca3a --name="$CLUSTER_NAME"
```

Comment:
suggestion: same again
```suggestion
$KIND_BIN create cluster --image=kindest/node@sha256:100b3558428386d1372591f8d62add85b900538d94db8e455b66ebaf05a3ca3a --name="$CLUSTER_NAME"
```